server.js

```diff
@@ -5728,6 +5728,62 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Opera add-ons integration
+camp.route(/^\/opera-add-ons\/(v|d|rating|stars|rating-count)\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  const type = match[1];
+  const name = match[2];
+  const format = match[3];
+  const badgeData = getBadgeData('opera add-ons', data);
+  const url = 'https://addons.opera.com/en/extensions/details/' + name;
+  request(url, function(err, res, buffer) {
+    if (!err) {
+      try {
+        const cheerio = require('cheerio');
+        const $ = cheerio.load(buffer);
+        const version = $('section.about > dl > dd').eq(2).text();
```

Comment:
Something as simple as Opera switching the order of the fields in the "about the extension" section (for example, displaying "size" above "version") will break this. If you really do need to parse the HTML, it might be worth making it smarter (eg. look for the `<dt>` containing "Version" and then look in the `<dd>` directly after that)

I'd suggest pulling this out into a separate utility module (eg. `lib/parseOperaExtensionPage.js`) rather than putting all the code in here.